components/SearchSandbox/index.js

```diff
@@ -70,6 +70,7 @@ class SearchSandbox extends Component {
 						this.setState({
 							profileList,
 							loading: false,
+							analytics: this.pref.analytics,
```

Comment:
For users who is not having analytics in body response. How this will work?

Reply:
```suggestion
							analytics: this.pref.analytics || true,
```
and also its handled in `localPref`

Reply:
Can you please commit this ?